src/deploy-tes-on-azure/scripts/helm/values-template.yaml

```diff
@@ -19,6 +19,15 @@ config:
     usePreemptibleVmsOnly: RUNTIME_PARAMETER
     poolRotationForcedDays: "7"
     prefix: RUNTIME_PARAMETER
+  deployment:
+    organizationName: RUNTIME_PARAMETER
+    organizationUrl: RUNTIME_PARAMETER
+    contactUri: RUNTIME_PARAMETER
+    environment: RUNTIME_PARAMETER
+    created: RUNTIME_PARAMETER
+    updated: RUNTIME_PARAMETER
+  nodeImages:
+    docker: RUNTIME_PARAMETER
```

Comment:
`nodeImages.docker` is no longer used. It should not be restored. If we're still setting it anywhere else, it should be removed.

Reply:
I addressed this